gitlab.go

```diff
@@ -187,9 +187,6 @@ func (c *Client) SetBaseURL(urlStr string) error {
 		return err
 	}
 
-	// Set the encoded opaque data
-	c.baseURL.Opaque = c.baseURL.Path
-
```

Comment:
OK, I reverted back Go 1.5 related changes. I just made one small change in this Opaque related workaround.
I moved the following logic from SetBaseURL
```
c.baseURL.Opaque = c.baseURL.Path
```
to the newRequest method:
```
u.Opaque = c.baseURL.Path + path
```
So in such case baseUrl variable is not affected by this workaround and BaseURL method always return full url: 
```
https://gitlab.com/api/v3/ 
```
instead of: 
```
https:/api/v3/
```